packages/lesswrong/components/editor/EditorTypeSelect.tsx

```diff
@@ -28,7 +28,7 @@ const EditorTypeSelect = ({value, setValue, isCollaborative, classes}: {
     }
   });
   
-  if (!currentUser?.reenableDraftJs && !currentUser?.isAdmin) return null
+  if (!currentUser?.markDownPostEditor && !currentUser?.isAdmin) return null
```

Comment:
After some consideration, Ruby and I decided to make it so that Markdown users see the EditorTypeSelect UI.

Previously, people who had the draft-js editor enabled saw the version control (I think the reason some markdown users have draft-js enabled is just to give them more control over what editor they're viewing)

If we didn't enable this, then the only way to switch back and forth between markdown and ckEditor would be to go over to your user settings, toggle "markDownPostEditor", and then go back to your post and click "set to your default editor". 

It's possible we should change the way people interact with markdown and not give them the affordance to switch back and forth, but that's a kinda big change that requires more thought.